### DISCUSSION TITLE
How do we add completion from github page?

### DISCUSSION THREAD

**Discussion by karman011** [zdharma-continuum/zinit#610] [Q&A] [answered]
example
```
zinit ice as"command" from"gh-r" mv"bat* -> bat" pick"bat/bat"
zinit light sharkdp/bat
```
We can see `sharkdp---bat/bat/autocomplete/bat.zsh` exists. But how do we load it. Zinit can't automatically load it. From my observation zinit only automatically load completion like "_bat" but not "bat.zsh".

**Comment by pschmitt** [comment]
You can move the file (`mv"a -> _a"`)
zinit should pick it up then

**Reply by karman011** [reply]
Thanks for reply! I thought we can only use the move modifier once? Anyway I solved this by
```
zinit ice as"command" from"gh-r" mv"bat*/autocomplete/bat.zsh -> _bat" pick"bat*/bat"
zinit light sharkdp/bat
```
I can't rename the directory now but that's fine.
cp modifier would also work.

**Comment by vladdoster** [comment, marked as answer]
@karman011 @pschmitt,

PR #618 adds logic where Zinit will automatically do this for you.

<img width="1728" alt="Screenshot 2024-01-09 at 05 38 15" src="https://github.com/zdharma-continuum/zinit/assets/10052309/1482ddc2-2194-43b4-9697-35bd45bc2aaf">
